SpatialGDK/Source/SpatialGDKTests/Examples/SpatialGDKTestGuidelines.h

```diff
@@ -0,0 +1,56 @@
+/*
+0. How to run tests:
+	- Tests can be run via Session Frontend in UE4 Editor
+
+	- Tests can be run via command line
+
+1. Folder structure
+	- Tests are located in a separate SpatialGDKTests module.
```

Comment:
Does this mean we aren't testing things that aren't exported from the main module?

Reply:
You are right, we cannot test anything that's not exported.

Reply:
Is that not too restrictive?

Reply:
It should be fine. I'd rather solve this if it becomes a problem.

Reply:
Surely most classes aren't exported.

Reply:
We'll have to export them then.

Reply:
I also agree that is a limitation, that we can workaround with dllexport. Maybe we should get some more opinions.

Reply:
Updated.